ena-submission/src/ena_deposition/create_sample.py

```diff
@@ -193,6 +195,28 @@ def set_sample_table_entry(db_config, row, seq_key, config: Config):
         )
 
 
+def safe_update_sample(
+    db_config: SimpleConnectionPool,
+    condition: dict[str, str],
+    update_values: dict[str, Any],
+    retry_number: int = 3,
+    subject: str = "Sample update",
+) -> None:
+    update_with_retry(
+        db_config=db_config,
+        conditions=condition,
+        update_values=update_values,
+        table_name=TableName.SAMPLE_TABLE,
+        retry_number=retry_number,
+        subject=subject,
+        success_log_fmt="{subject} for accession {accession} version {version} and DB updated!",
```

Comment:
If I substitute the default `Sample updated` I get: `Sample update for accession {accession} version {version} and DB updated!` which doesn't read right